app/javascript/components/I18nRaw.jsx

```diff
@@ -1,8 +0,0 @@
-import React from 'react'
-export default class I18nRaw extends React.Component {
-  render() {
-    return (
-      <div dangerouslySetInnerHTML={{__html: I18n.t(this.props.t)}}/>
```

Comment:
Waarom is deze weg? Ik denk dat we dit nog wel kunnen gebruiken. Ik heb het nu ook ergens gebruikt voor bijv de disclaimer tekst in een ander PR.

Reply:
Ik had m weggehaald omdattie niet precies deed wat ik wilde en ik hem verder niet gebruik. Ik wilde eigenlijk de tekst rechtstreeks gebruiken, maar dit gooit er nog een div tag omheen. En dat ging fout omdat ik dit ding in een label wilde gebruiken en de layout ging daardoor raar doen.

Maar als jij m wel gebruikt laten we m gewoon staan. Ik zal er wel even specs voor moeten schrijven.

Reply:
Ik heb m teruggezet en er specs voor geschreven.